.changeset/violet-flowers-poke.md

```diff
@@ -0,0 +1,45 @@
+---
+"@latticexyz/store": patch
+"@latticexyz/world": patch
+---
+
+You can now configure resources (tables, systems, etc.) across multiple namespaces in a single MUD config using the `namespaces` option.
+
+Note that once you start using `namespaces` config option, you will need to move your existing tables, systems, etc. under the corresponding namespace within `namespaces`.
+
+```ts
+export default defineWorld({
+  namespaces: {
+    game: {
+      tables: {
+        Health: {
+          schema: {
+            player: "address",
+            value: "uint32",
+          },
+          key: ["player"],
+        },
+        Position: {
+          schema: {
+            player: "address",
+            x: "int32",
+            y: "int32",
+          },
+          key: ["player"],
+        },
+      },
+    },
+    somePlugin: {
```

Comment:
good to give folks a concrete example rather than "some" abstract one
```suggestion
    scoreboardPlugin: {
```

Reply:
that's the name i was looking for 🙇

Reply:
This namespace name is actually too long, it causes the project to fail in in unexpected ways

Reply:
we should fix that and/or add it to validation

Reply:
Working on it in https://github.com/latticexyz/mud/pull/2897